src/katgpucbf/xbgpu/engine.py

```diff
@@ -228,8 +239,7 @@ def __init__(
         context: AbstractContext,
         init_tx_enabled: bool,
```

Comment:
This parameter seems to be unused now.

Reply:
Is it showing up for you at the end of `self.send_stream = XSend(...)`?
* Arg is `tx_enabled=init_tx_enabled`